### DISCUSSION TITLE
Project Planning and Staying Connected

### DISCUSSION THREAD

**Discussion by oscarlevin** [OpenMathBooks/small-documents#5] [General]
Here are some additional online homes for continuing work on the PreTeXt for Small Documents project.

- Here (or wherever this repository ends up)
- The google groups: [PreTeXt-Announce](https://groups.google.com/g/pretext-announce), [PreTeXt-Support](https://groups.google.com/g/pretext-support), and [PreTeXt-Dev](https://groups.google.com/g/pretext-dev).  If you are not already, you should definitely subscribe to at least the first two.
- Discord: there is now a channel for [small documents](https://discord.gg/Sb5VATjAuy)
- Github repositories for pretext projects, including [PreTeXt](https://github.com/PreTeXtBook/pretext), the [CLI](https://github.com/PreTeXtBook/pretext-cli), and [pretext-tools](https://github.com/oscarlevin/pretext-tools).  Also, for documentation, [Jason's repository](https://github.com/siefkenj/pretext-book), and the one for [Doenet](https://github.com/Doenet/DoenetML).

I think that using existing communication channels like these is most likely to be successful, but we could also set up a Github Project to track progress on particular tasks (this would be a way to have work spanned across multiple github repositories.

Please feel free to post additional links, resources, and ideas below.

**Comment by sfrezza** [comment]
Oscar, Rob and the whole team - **Thank you** for your marvelous work, and I'm very thankful to find out, and hopefully find a way to participate.

**Reply by sfrezza** [reply]
PS: I'd love to get an exemplar template of a small document using doenet for _binary_ math. Who would be the right person to connect with?
